.github/ISSUE_TEMPLATE.md

```diff
@@ -8,12 +8,6 @@
 <!--- Se você está descrevendo um _bug_, conte-nos o que deveria acontecer. -->
 <!--- Se você está sugerindo uma mudança/melhoria, conte-nos como deve funcionar ou como deve ser. -->
 
-## Current Behavior
```

Comment:
Acho ruim essa remoção, como será indicados os comportamentos dos bugs?

Reply:
We saw this release that `feature` and `improvement` are not the same thing and we feed that is necessary to create another type of branch tag

Reply:
The first part of the issue is the `Problem description` in which it is exactly meant to describe what is going wrong. That way `Current Behavior` was duplicated with `Problem description`

Reply:
Do you agree about this @sconetto?

Reply:
It can be inferred, but I don't see as a duplicate. However, if the new model will help with the evolution of SIGS, I'm on board!